device-types/Lenovo/SR850.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: Lenovo
+model: ThinkSystem SR850
+slug: sr850
+part_number: 7B27A03953
+u_height: 2
+is_full_depth: true
+comments: '[Lenovo System SR850 product guide](https://lenovopress.lenovo.com/lp1052.pdf)'
+subdevice_role: parent
+interfaces:
+  - name: XCC
+    type: 1000base-t
+    mgmt_only: true
+power-ports:
```

Comment:
According to the datasheet, the PSUs are hot-swappable. So the power slots should be created in `module-bays`, and then the associated PSU's themselves should be added to the module-types folder if they do not already exist